### PR TITLE
common.config: Cleanup and add a lot of Packages

### PR DESCRIPTION
**Pull request by stargieg**
This is the min Package selection that I need for my Freifunk Routers.

### CODE REVIEW

**Comment by SvenRoederer** [comment]
what about adding the openwrt-repository to your sources and using the app-packages from there?

**Comment by stargieg** [comment]
I can load a kmod from openwrt Build into a Kernel from Freifunk Build bot?

**Comment by SvenRoederer** [comment]
That's why I was referring to the app-packages for the applications.
Kernel-packages will for sure maybe only work with some "force" options

File: configs/common.config [hunk 1]
```diff
@@ -163,7 +158,6 @@ CONFIG_PACKAGE_uhttpd-mod-ubus=m
 CONFIG_PACKAGE_vnstat=m
 CONFIG_PACKAGE_wpad=m
 CONFIG_PACKAGE_zlib=m
-CONFIG_SDK=y
```
Comment (SvenRoederer) [inline configs/common.config]: see issue #337, we should keep this

**Pull request closed without merging**